db/config.cc

```diff
@@ -871,6 +871,9 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , enable_sstables_mc_format(this, "enable_sstables_mc_format", value_status::Unused, true, "Enable SSTables 'mc' format to be used as the default file format.  Deprecated, please use \"sstable_format\" instead.")
     , enable_sstables_md_format(this, "enable_sstables_md_format", value_status::Unused, true, "Enable SSTables 'md' format to be used as the default file format.  Deprecated, please use \"sstable_format\" instead.")
     , sstable_format(this, "sstable_format", value_status::Used, "me", "Default sstable file format", {"md", "me"})
+    , uuid_sstable_identifiers_enabled(this,
+            "uuid_sstable_identifiers_enabled", value_status::Used, false, "If set to true, each newly created sstable will have a UUID "
```

Comment:
Should we make it live-updatable? I can't think of a reason why.

But, let's default it to true (otherwise, we do have to make it live-updatable).

Reply:
changed. and fingers crossed..

Reply:
This patch is to be dropped, yes?

Reply:
yeah, sorry. i failed to squash it. dropped.

Reply:
@avikivity thank you for your ideas. they are really helpful!

> I wanted it dropped, not squashed. I don't understand the problems with sstableloader. Maybe it's trying to load files that were generated by scylladb itself and so it doesn't understand the new format:
> 
> Ideas for workaround:
> 
>     1. disable the config item just for those test
> 

i am taking this approach: https://github.com/scylladb/scylla-dtest/pull/3220

>     2. adjust the tests to rename the files to have integer generations
> 
> 
> But I'm okay with merging it with it disabled and enabling it soon.